src/js/other-websites/inpage_menu.js

```diff
@@ -159,8 +192,57 @@ async function populateMaskList(
   list.classList.remove("is-loading");
 
   maskList.classList.add("is-visible");
-  
-  // The free UI has both lists wrapped up in a hidden-by-default element, so this makes it visible too. 
+
+  // The free UI has both lists wrapped up in a hidden-by-default element, so this makes it visible too.
+  document.querySelector(".fx-relay-menu-masks-lists")?.classList.add("is-visible");
+
+  await browser.runtime.sendMessage({
+    method: "updateIframeHeight",
+    height: document.getElementById("fxRelayMenuBody").scrollHeight,
+  });
+}
+
+async function populatePremiumMaskList(maskList, masks) {
+  const list = maskList.querySelector("ul");
+
+  if (masks.length === 0) {
+    return;
+  }
+
+  masks.forEach((mask) => {
+    const listItem = document.createElement("li");
```

Comment:
suggestion (non-blocking) Is is still considered faster to use `createDocumentFragment` instead of `createElement`?

Reply:
Replaced for the `li` `createElement`. The other element (`button`) is still using the other method, but is being injected into the fragment. Should be good to go. 😎 

Revised in d5e6221fe717faa6809d549a9e0b85259f67a063